fiftyone/server/db/_mongo_lightning.py

```
@@ -0,0 +1,627 @@
+
+
+def _add_search(query: DistinctQuery):
+    # strip chars after 24
+    if query.is_object_id_field:
+        search = query.search[:_TWENTY_FOUR]
+        add = (_TWENTY_FOUR - len(search)) * "0"
+        if add:
+            search = f"{search}{add}"
+        try:
+            value = {"$gte": ObjectId(search)}
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)